addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -705,8 +707,14 @@ def _compose_multi_info_message(self, tree, element_tags):
     @api.model
     def _get_xml_decoders(self):
         # Override
-        ubl_decoders = [('Italy', self._detect_italy_edi, self._import_xml_invoice)]
-        return super(AccountMove, self)._get_xml_decoders() + ubl_decoders
+        if not self.journal_id.edi_ids.filtered(lambda e: e.name == 'FatturaPA'):
```

Comment:
we need something more generic I think. Maybe a code on `account.edi` and retrieve the decoders based on this?
For example, `code=ubl` and make something like 

```
for edi in self.journal_id.edi_ids:
  decoders.add(getattr('_get_%s_decoder' % edi.code))
```

to discuss

Reply:
As discussed, we will use hooks in combination with `code == ubl`